### DISCUSSION TITLE
How to convert require into import statement in Typescript ?

### DISCUSSION THREAD

**Discussion by ishyR** [cypress-io/cypress#25302] [General]
Hi,
I'm a beginner in Cypress Automation. How do I convert below JAVASCRIPT statement to TYPESCRIPT? At the moment, I'm getting an error.

Much appreciate your help ! 

**Error Message**: Require statement not part of import statement
**Statement in 'command.ts'**: require('cy-verify-downloads').addCustomCommand();"

![image](https://user-images.githubusercontent.com/121594259/209902507-59d10052-0bcf-47ea-a942-2f758343126d.png)

**Comment by SAhmed1713** [comment]
Have anyone find any solution for the above mentioned problem?. I am facing the same problem with my cypress.config.ts file.
